desktop/release-notes.md

```diff
@@ -33,6 +33,60 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/ge
 
 ### New
 
+- Introduce the new Dev Environments CLI `docker dev`.
+- Docker Desktop can now be installed to any drive and folder using the `--installation-dir`. Partially addresses [docker/roadmap#94](https://github.com/docker/roadmap/issues/94).
+- Implemented enhanced-container-isolation feature as part of Docker Desktop's hardening-desktop effort (public link to be provided).
+
+### Upgrades
+
+- [Docker Scan v0.21.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.21.0)
+- [Go 1.19.2](https://github.com/golang/go/releases/tag/go1.19.2) to address [CVE-2022-2879](https://www.cve.org/CVERecord?id=CVE-2022-2879){: target="_blank" rel="noopener"}, [CVE-2022-2880](https://www.cve.org/CVERecord?id=CVE-2022-2880){: target="_blank" rel="noopener"} and  [CVE-2022-41715](https://www.cve.org/CVERecord?id= CVE-2022-41715){: target="_blank" rel="noopener"}
+- [Docker Engine v20.10.18](https://docs.docker.com/engine/release-notes/#201018), which contains a fix for [CVE-2022-36109](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-36109).
+- [Amazon ECR Credential Helper v0.7.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.7.0){: target="blank" rel="noopener" class=""}
+- [Docker Compose v2.11.2](https://github.com/docker/compose/releases/tag/v2.11.2)
+- [Kubernetes v1.25.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.25.2)
+- [Qemu 7.0.0](https://wiki.qemu.org/ChangeLog/7.0)
+- [Linux kernel 5.15.49](https://hub.docker.com/layers/docker/for-desktop-kernel/5.15.49-13422a825f833d125942948cf8a8688cef721ead/images/sha256-ebf1f6f0cb58c70eaa260e9d55df7c43968874d62daced966ef6a5c5cd96b493?context=explore)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Allow the use of TLS when talking to HTTP and HTTPS proxies to encrypt proxy usernames and passwords.
+- Added a user setting to opt out of 4.13-specific features that are in development.
+- Fixed a bug that prevented access to the Docker Desktop VM through the console/debugging port when the enhanced-container-isolation feature is turned on.
```

Comment:
```suggestion
- Fixed a bug that prevented access to the Docker Desktop VM through the console/debugging port when the Enhanced Container Isolation feature is turned on.
```

Reply:
Do we need to mention this fix, since Enhanced Container Isolation feature is only being introduced in this release?

Reply:
oh, I'll delete it.